lib/src/rules/prefer_string_spaces_at_beginning.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:collection';
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = "Put spaces in wrapped strings on the beginning of new lines.";
+
+const _details = '''
+
+**DO** put your spaces between words in wrapped strings at the beginning of new
+lines.
+
+That means you should prefer '... foo' \\n ' bar' over '... foo ' \\n 'bar'.
+This provides a few advantages:
+
+- with this lint on, you can't accidentally do *both*.
+- with this lint on, your project will at least be consistent in either way
+- by preferring spaces at the beginning, it is easier for code reviewers to
+  catch when they are missing entirely
+
+Pretty much every exception to this is in raw strings, which is a solid
+indicator of something that is not human text and does not conform to any
+predictable spacing pattern.
+
+**BAD:**
+```
+ 'some amount of words that wrap around where the space is very '
+ 'easily missed'
+```
+
+**GOOD:**
+```
+ 'some amount of words that wrap around where the space is very'
+ ' easily recognized'
+```
+
+''';
+
+class PreferStringSpacesAtBeginning extends LintRule {
+  _Visitor _visitor;
+
+  PreferStringSpacesAtBeginning()
+      : super(
+            name: 'prefer_string_spaces_at_beginning',
+            description: _desc,
+            details: _details,
+            group: Group.style) {
+    _visitor = new _Visitor(this);
+  }
+
+  @override
+  AstVisitor getVisitor() => _visitor;
+}
+
+class _Visitor extends SimpleAstVisitor<bool> {
+  final LintRule rule;
+  final Set<AstNode> visited = new HashSet<AstNode>();
```

Comment:
Not sure why this set is necessary. Please add documentation (assuming you keep it).